build_tools/bazel/iree.bazelrc

```
@@ -4,6 +4,14 @@
 # See https://llvm.org/LICENSE.txt for license information.
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
+###############################################################################
+# Short-hand config settings for IREE specific features.
+###############################################################################
+
+# Equivalent to CMake flag IREE_ENABLE_RUNTIME_TRACING.
+# Builds the runtime with tracing support enabled.
+build:iree_enable_runtime_tracing --@tracy_client//:runtime_enable=true
```

Rather than a config, you can use a flag alias. I don't think the explicit "=true" is necessary either.

```suggestion
build --flag_alias=iree_enable_runtime_tracing=@tracy_client//:runtime_enable
```

---

(Please remember to also update the description)

---

Ah - I didn't know about that! Thanks.